frontend/src/components/widget/Attributes/Attributes.js

```diff
@@ -220,24 +220,30 @@ export default class Attributes extends Component {
         value,
       }).then((response) => {
         if (response.data && response.data.length) {
-          const fields = response.data[0].fieldsByName;
-
-          Object.keys(fields).map((fieldName) => {
-            this.setState(
-              (prevState) => ({
-                data: {
-                  ...prevState.data,
-                  [fieldName]: {
-                    ...prevState.data[fieldName],
-                    value,
-                  },
-                },
-              }),
-              () => {
-                cb && cb();
-              }
-            );
+          const { fieldsByName, id } = response.data[0];
+          const hasCheckbox = JSON.stringify(fieldsByName).includes('YesNo');
```

Comment:
why checking for checkboxes?
i mean, we could change the display logic of other fields on any field change, not only  on checkboxes.

Reply:
Yes, but right now there's no other way for me to know whan can trigger an update that changes layout or not. This request should just trigger a websocket event but since we don't have it now I had to patch something on the client side.

Reply:
basically, anything can trigger a layout change.

But, when frontend is patching something it gets back the changed fields.
Some of them have the "displayed" flags changed.
That might  be an indicator of what have changed....

Reply:
Yeah, you're right. This seems to be good enough to use it.